### PR TITLE
Added a workaround for a bug with Chrome for CSS position:sticky

### LINKED ISSUE #97: The height of the welcome section should be equal to the height of the viewport. test fails
#### Issue Description
I've done Portfolio challenge.
And even tho I've asked around and looked at example,
to see what I did wrong looks like I did nothing wrong,
yet fcc validator doesn't seem to let me pass.

#### Browser Information
* Browser Name, Version: Google Chrome
* Operating System: Windows 10
* Mobile, Desktop, or Tablet:  Desktop

#### Your Code / Link to Your Pen
http://codepen.io/diomed/pen/EmdMrb
```
 
 
 
```

### PR DESCRIPTION
**Pull request by tchaffee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXX with an issue no): Closes #97 

#### Description

This test was failing due to [a documented bug in Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=672457). 

In short, CSS `position: sticky` can cause `getBoundingClientRect()` to report the wrong values if executed immediately after `window.scroll`. The workaround is to use `setTimeout` to yield to the browser to allow page layout to complete before running `getBoundingClientRect()`.

### CODE REVIEW

**Comment by tchaffee** [comment]
@tbushman Good point about the bundle.js. That would be good to research before we merge any of our PRs. I can try to look into that tomorrow by looking at older PRs. If we don't find something similar we may want to hold off on merging these until we understand why that is happening.

File: src/project-tests/portfolio-tests.js [hunk 1]
```diff
@@ -111,12 +111,21 @@ export default function createPortfolioTests() {
...
+                // the getBoundingClientRect can sometimes report the wrong value while the page is
+                // still laying out, when using CSS position:sticky.
+                // This is apparently a bug with Chrome https://bugs.chromium.org/p/chromium/issues/detail?id=672457
+                setTimeout(function () {
+                  FCC_Global.assert.approximately(navbar.getBoundingClientRect().top, 0, 15, 'Navbar should be at the top of the viewport even after scrolling ');
+                  window.scroll(0, 0);
+                  done();
+                }, 1);
+
             });
```
Comment (no-stack-dub-sack) [inline src/project-tests/portfolio-tests.js]: @tchaffee Cool! Nice find on this one! Did you test your solution against the failing code reported in the original issue?

Also, looks like we're still seeing all the jquery update changes in this PR - shouldn't be an issue but is making the diff hard to parse.

Comment (tchaffee) [inline src/project-tests/portfolio-tests.js]: @no-stack-dub-sack Yes, this was tested against the failing code, and fixes it. Concerning the jquery changes, once I rebase over master they should go away. I can do that today.

Comment (tchaffee) [inline src/project-tests/portfolio-tests.js]: @no-stack-dub-sack I think this should be ok to merge now?

**Comment by no-stack-dub-sack** [comment]
### LGTM.
